config/jobs/kubeflow/kubeflow-presubmits.yaml

```diff
@@ -198,3 +198,83 @@ presubmits:
       containers:
       - image: gcr.io/kubeflow-ci/test-worker:latest
         imagePullPolicy: Always
+
+  kubeflow/pipelines:
+  - name: build-image
+    always_run: true
+    decorate: true
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20181024-be2f242dd-1.10
+        command:
+        - ./test/presubmit-tests.sh
+        args:
+        - --workflow_file
+        - build_image.yaml
+    run_after_success:
+    - name: presubmit-e2e-test
+      always_run: true
+      decorate: true
+      labels:
+        preset-service-account: "true"
+      spec:
+        containers:
+        - image: gcr.io/k8s-testimages/kubekins-e2e:v20181024-be2f242dd-1.10
+          command:
+          - ./test/presubmit-tests.sh
+          args:
+          - --workflow_file
+          - e2e_test_gke.yaml
+          - --test_result_folder
+          - e2e_test
+    - name: presubmit-e2e-test-gce-minikube
+      always_run: true
+      decorate: true
+      skip_report: true
+      labels:
+        preset-service-account: "true"
+      spec:
+        containers:
+        - image: gcr.io/k8s-testimages/kubekins-e2e:v20181024-be2f242dd-1.10
+          command:
+          - ./test/presubmit-tests-gce-minikube.sh
+          args:
+          - --workflow_file
+          - e2e_test_gke.yaml
+          - --test_result_folder
+          - e2e_test_gke
+    - name: presubmit-sample-test
+      always_run: true
+      decorate: true
+      skip_report: true
```

Comment:
Will there be rerun command that simply trigger one of the test?

Reply:
rerun command is default in latest prow now, aka `/test $JOB_NAME` will trigger a rerun

Reply:
I believe Sen mentioned the command is available by default as  
/test [test-name]

Reply:
awesome. thanks